### ISSUE TITLE
Synchronous functionality when passing raw html

### ISSUE THREAD

**Issue by krazyjakee** [luin/readability#91]
There is no reason the parser should run asynchronously when passing raw html as there is no request to be made.

Usage should be something like...
`{ err, article, meta } = read(rawHtml)`

**Comment by mhamann** [comment]
Not true. The read/parse logic could be CPU intensive and block for quite awhile. With promises and async/await, it shouldn't make much difference, except for not blocking the main thread (which is a good thing!).

**Comment by krazyjakee** [comment]
@mhamann doh! I should have thought about that. I would be satisfied with a promise solution too but then may as well do a total ES8 rewrite. I don't have time for it unfortunately.
